what4/src/What4/Solver/CVC5.hs

```diff
@@ -244,3 +250,124 @@ instance OnlineSolver (SMT2.Writer CVC5) where
           timeout feat (Just cvc5StrictParsing) mbIOh sym
 
   shutdownSolverProcess = SMT2.shutdownSolver CVC5
+
+
+data CVC5_SyGuS = CVC5_SyGuS deriving Show
```

Comment:
As far as I can tell, the only difference between `CVC5_SyGuS` and `CVC5` is the set of command-line flags that get passed to the `defaultSolverArgs` in their `SMTLib2GenericSolver` instances. Is this correct? If so, this would be worth documenting.